taxi/forms.py

```diff
@@ -0,0 +1,56 @@
+from django import forms
+from django.contrib.auth import get_user_model
+from django.contrib.auth.forms import UserCreationForm
+from django.core.exceptions import ValidationError
+from django.forms import ModelForm
+from django.http import HttpResponse
+
+from taxi.models import Driver, Car
+
+
+class DriverForm(UserCreationForm):
+    class Meta:
+        model = Driver
+        fields = UserCreationForm.Meta.fields + (
+            "first_name",
+            "last_name",
+            "license_number"
+        )
+
+
+class DriverLicenseUpdateForm(forms.ModelForm):
+    class Meta(UserCreationForm.Meta):
+        model = Driver
+        fields = ("license_number",)
+
+    def clean_license_number(self) -> HttpResponse:
+        license_number = self.cleaned_data["license_number"]
+        if len(license_number) != 8:
+            raise ValidationError(
+                "License number should be 8 characters long"
+            )
+        if not license_number[:3].isupper():
+            raise ValidationError(
+                "First three license number characters should be upper case"
+            )
+        if not license_number[3:].isdigit():
+            raise ValidationError(
+                "Last five license number should be digits only"
+            )
+        if not license_number[:3].isalpha():

```

Comment:
The redundant check for license_number[:3].isalpha() overlaps with the earlier uppercase letter check. Simplifying these validations is essential for clarity and efficiency.